tests/nosetests/test_blackbody/test_blackbody.py

```diff
@@ -31,14 +31,16 @@ def test_binary(plot=False, gen_comp=False):
     b.set_value_all('ld_func', 'linear')
     b.set_value_all('ld_coeffs', [0.0])
 
-    #turn off albedos (legacy requirement)
+    # turn off albedos (legacy requirement)
     b.set_value_all('irrad_frac_refl_bol',  0.0)
 
+    b.set_value_all('blending_method', 'none')
+
     if plot: print("running phoebe2 model...")
-    b.run_compute(compute='phoebe2', irrad_method='none', model='phoebe2model')
+    b.run_compute(compute='phoebe2', irrad_method='none', model='phoebe2model', blending_method='none')
```

Comment:
these are redundant, but shouldn't hurt either

Reply:
I was overwriting the value while testing, and this work continues, that's why I left it in there.

Reply:
Does it work with and without line 37?  It might be nice to test a case each way to make sure that the backend is accepting the in-line kwarg override as expected.

Reply:
I'm inclined to defer that test to the time when extrapolation is fully ported to C.